ksql-engine/src/test/java/io/confluent/ksql/QueryTranslationTest.java

```diff
@@ -367,7 +368,7 @@ private static Topic createTopicFromStatement(final String sql) {
       } else {
         avroSchema = Optional.empty();
       }
-      return new Topic(topicName, avroSchema, getSerdeSupplier(format), 1);
+      return new Topic(topicName, avroSchema, getSerdeSupplier(format), KsqlConstants.defaultSinkNumberOfPartitions);
```

Comment:
We should limit the use of these constants in tests to those testing the legacy path is doing the right thing.  Everywhere else we should just hard code the right value. 

Like here - the QTT test doesn't need to set the partition count to the legacy default here - the QTT tests should be using the _new_ functionality unless the test case explicitly disables it. 

Using the constant here is misleading.

Can you look at all uses of `defaultSinkNumberOfPartitions` and the similar one for replica count and remove any where the test is not explicitly testing the legacy code path, please?